src/components/blocks/index.ts

```diff
@@ -0,0 +1,32 @@
+import { TextBlock } from "./TextBlock";
+import { NumberBlock } from "./NumberBlock";
+import { SelectBlock } from "./SelectBlock";
+import { RichTextBlock } from "./RichTextBlock";
+import { DatePickerBlock } from "./DatePickerBlock";
+import { MultipleCheckboxesBlock } from "./MultipleCheckboxesBlock";
+import { BooleanBlock } from "./BooleanBlock";
+import type { FC } from "react";
+
+export const blocksMap = {
+    text: TextBlock,
+    phonenumber: TextBlock,
+    radio: TextBlock,
+    html: RichTextBlock,
+    textarea: RichTextBlock,
+    number: NumberBlock,
+    date: DatePickerBlock,
+    checkbox: MultipleCheckboxesBlock,
+    select: SelectBlock,
+    booleancheckbox: BooleanBlock,
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+} as Record<string, FC<any>>;
```

Comment:
No need to type as Record here. Instead it should be `as const` as it doesn't change at all.

Reply:
Fixed as `export const blocksMap = { ... } as const as BlocksMap`